### DISCUSSION TITLE
Ruler Generator

### DISCUSSION THREAD

**Discussion by Robbbb** [microsoft/maker.js#514] [Ideas]
Hey Makers!

Many years ago, with a lot of help, I built a [Ruler Generator ](https://github.com/Robbbb/VectorRuler) in paper.js.
Maker.JS is way better for this application, and I would love to see it in a more feature-rich and sustainable home.

Sadly, I don't have the skills or support to make tis happen.
I think it would take very little time for someone with experience, as much of the code may be portable. 
I tried to get started, [details are here. ](https://github.com/Robbbb/VectorRuler/issues/26)

Your help would make a big difference for the users of the generator.

**Comment by Robbbb** [comment]
@danmarshall How do I add this to the demos board in Projects?
Any tips on making this happen?
I offer to mail a laser-cut custom ruler to anyone who takes this on!

**Reply by danmarshall** [reply]
I think only admins can modify project boards, so I've added this for you 👍🏽 
I'm hoping that someone can take this task on, as I've got other previous commitments in the core functionality.

**Reply by Robbbb** [reply]
Hey @danmarshall! any chance you are interested? I would be happy to send you any laser cut thing as a reward! 🙂

**Reply by danmarshall** [reply]
Hi @Robbbb this is cool, but I've got other commitments right now, cheers!
